core/server/web/middleware/url-redirects.js

```diff
@@ -26,7 +26,7 @@ _private.redirectUrl = function redirectUrl(options) {
     });
 };
 
-_private.getAdminRedirectUrl = function getAdminRedirectUrl(options) {
```

Comment:
Was thinking if `_private` should be left alone and the export to be done with `module.export.getAdminRedirectUrl = _private.getAdminRedirectUrl;` instead. But as these are no longer strictly private methods decided to change it to avoid confusing notation.

Reply:
Yeah probably makes sense 👍